### PR TITLE
Grid

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: components/grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.two-column-grid{
+    width: 30%;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
```
Comment (yaswanthmyneni) [inline components/grid/grid.css:3]: changes: give css variables for colors

File: components/grid/grid.css [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+}
+
+.content-container button{
+    cursor: pointer;
+    font-size: 2rem;
+    padding: .5rem 1rem;
```
Comment (yaswanthmyneni) [inline components/grid/grid.css:23]: changes: give px units for less than 1rem

File: components/grid/grid.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    border: none;
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
```
Comment (yaswanthmyneni) [inline components/grid/grid.css:27]: changes: same here and everywhere, use css variables

File: components/toast/toast.css [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 1px 3px 1px;
+    color: var(--primary-color);
+    border: none;
+    cursor: pointer;
+    padding: 1rem;
+    border: 1px solid var(--black-color);
+    margin-right: 1rem;
+}
+
+#baseline-toast{
```
Comment (yaswanthmyneni) [inline components/toast/toast.css:13]: changes: use classes for styling, avoid using id's for styling

File: components/toast/toast.css [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+    visibility: hidden;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--primary-color);
+    box-shadow: 0px 6px 10px gray;
```
Comment (yaswanthmyneni) [inline components/toast/toast.css:19]: changes: use css variables for colors

**Review by yaswanthmyneni — APPROVED** [review]
replace all id's with classes for styling, and use px for less than 1rem, and use css variables for colors. Merge after changes

File: components/grid/grid.css (hunk 1 above)
Comment (vermasavita) [reply components/grid/grid.css:3]: Noted

File: components/grid/grid.css (hunk 2 above)
Comment (vermasavita) [reply components/grid/grid.css:23]: Noted. Changed

File: components/toast/toast.css (hunk 4 above)
Comment (vermasavita) [reply components/toast/toast.css:13]: Noted

**Review by prajapati-priyanka — APPROVED** [review]
Changes has already been suggested by the other reviewer. Work on that. Rest everything is good.
Good to merge.

**Pull request merged**